apps/freeablo/faworld/player.h

```diff
@@ -44,15 +44,15 @@ namespace FAWorld
 
         bool castSpell(SpellId spell, Misc::Point targetPoint) override;
         void setActiveSpellNumber(int32_t spellNumber);
+        SpellId getActiveSpell() const { return mActiveSpell; }
+        void setActiveSpell(SpellId spell) { mActiveSpell = spell; }
```

Comment:
When there is both a getter and setter with no special logic, I think it makes more sense to just make the variable public.